src/asciidoctor-kroki.js

```diff
@@ -44,6 +44,8 @@ const processKroki = (processor, parent, attrs, diagramType, diagramText, contex
   }
   if (diagramType === 'vegalite') {
     diagramText = require('./preprocess.js').preprocessVegaLite(diagramText, context)
+  } else if (diagramType === 'plantuml') {
+    diagramText = require('./preprocess').preprocessPlantUML(diagramText, context)
```

Comment:
Just a nitpick but I believe it will be easier to migrate to ESM if we have explicit extension:
```suggestion
    diagramText = require('./preprocess.js').preprocessPlantUML(diagramText, context)
```

(and it's consistent with the require just above)

Reply:
done